src/state/eventStateProvider.js

```diff
@@ -275,13 +275,13 @@ class EventsContainer extends Container {
   // allEvents will refresh all events (ie: from the index page), whereas setting it to false will refresh the interested event
   toggleInterest = async (event, singleEvent = false) => {
     const {user_is_interested, id} = event
+    const {remove, create} = server.events
```

Comment:
This is the wrong endpoint, you are trying to create and actual event not toggling the favorite.
```suggestion
    const {remove, create} = server.events.interesets
```